src/main/java/org/mskcc/cbio/portal/model/StructuralVariant.java

```diff
@@ -468,4 +468,58 @@ public String getAnnotationJson() {
     public void setAnnotationJson(String annotationJson) {
         this.annotationJson = annotationJson;
     }
+
+    @Override
+    public String toString() {
```

Comment:
why all fields, and not just some key fields?

Reply:
I always thought it was a common practice for `toString()` to include everything. Unlike `hashCode()` and `equals(o)` which use only key fields.

Which fields do you consider key fields?